sfyra/pkg/tests/reset.go

```diff
@@ -48,7 +48,7 @@ func TestServerReset(ctx context.Context, metalClient client.Client, vmSet *vm.S
 			require.NoError(t, err)
 		}
 
-		err = retry.Constant(10*time.Minute, retry.WithUnits(10*time.Second)).Retry(func() error {
+		err = retry.Constant(5*time.Minute, retry.WithUnits(10*time.Second)).Retry(func() error {
```

Comment:
There is no reason that a reset of a single server should take more than a minute let alone five. This will just keep CI from hanging.